app/models/manageiq/providers/openstack/cloud_manager/provision/cloning.rb

```diff
@@ -1,4 +1,12 @@
 module ManageIQ::Providers::Openstack::CloudManager::Provision::Cloning
+  def find_destination_in_vmdb(ems_ref)
+    vm_model_class.find_by(:ems_id => source.ext_management_system.id, :ems_ref => ems_ref)
+  rescue NoMethodError => ex
+    _log.debug("Unable to find Provison Source ExtmanagementSystem: #{ex}")
+    _log.debug("Trying use attribute src_ems_id=#{options[:src_ems_id].try(:first)} instead.")
+    vm_model_class.find_by(:ems_id => options[:src_ems_id].try(:first), :ems_ref => ems_ref)
```

Comment:
I wonder if we can just use this call right away? Or is there a case when options[:src_ems_id] is nil?

Reply:
I'm not aware of such case, but I'd prefer keep original method using _source_ to feel safe..

Reply:
right, it bugs me a bit that this will basically fail everytime for volume_template, so maybe we should not use it?